js/ez-toolbar.js

```diff
@@ -1,9 +1,29 @@
+
+/** JS code for the `ez-toolbar` custom element defined in ez-toolbar.html.
```

Comment:
please, look at #2 on how to do that properly... You are supposed to document the class corresponding to your custom element and to describe the element not the JS file...